wallet/udb/common_test.go

```diff
@@ -25,7 +24,9 @@ var (
 		0xef, 0x8d, 0x64, 0x15, 0x67,
 	}
 
-	emptyDb = "empty.kv"
+	emptyDb     = "empty.kv"
+	emptyDbPath = ""
+	testDir     = "udb-"
```

Comment:
this is only used by TestMain so let's move it out of global scope (could just pass a string literal to TempDir)